src/search-modal/SearchUI.test.tsx

```diff
@@ -487,4 +487,39 @@ describe('<SearchUI />', () => {
       expect(queryByLabelText(/^Lightcast/i)).toBeInTheDocument();
     });
   });
+
+  describe('searchKeywords', () => {
+    let rendered: RenderResult;
+    beforeEach(async () => {
+      rendered = render(<Wrap><SearchUI {...defaults} /></Wrap>);
+      // Wait for initial search request
+      await waitFor(() => { expect(fetchMock).toHaveFetchedTimes(1, searchEndpoint, 'post'); });
+    });
+
+    it('should update search results when keywords are entered', async () => {
+      const { getByRole, getByText } = rendered;
+      const searchBox = getByRole('searchbox');
+      fireEvent.change(searchBox, { target: { value: 'test search' } });
+      await waitFor(() => {
+        expect(fetchMock).toHaveLastFetched((_url, req) => {
+          const requestData = JSON.parse(req.body?.toString() ?? '');
+          // Check both the first and second query.q
+          return requestData?.queries[0]?.q === 'test search'
+            && requestData?.queries[1]?.q === 'test search';
+        });
+      });
+      expect(getByText('6 results found')).toBeInTheDocument();
```

Comment:
@DanielVZ96 These tests are great! But I think we should also add tests that verify the results of the block type query, so that if the search keywords change, the block type query also changes. Is that possible?